File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    let totalSum = 0;
    let totalMul = 1;
    for(let i of num_list){
        totalSum+= i
        totalMul*= i
    }
    
    if(totalMul<totalSum**2){
        return 1
    }
    else return 0
    
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
import java.util.*;

class Solution {
    public int solution(int[] num_list) {
        int A = num_list[0];
        int B = num_list[0];
        
        for(int i =1; i<num_list.length; i++){
            A = A * num_list[i];
            B += num_list[i];
        }
        
        System.out.println(A);
        System.out.println(B);
        return A < (int)Math.pow(B,2) ? 1 : 0;
    }
}